roles/ceph-mon/tasks/deploy_monitors.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: check if monitor initial keyring already exists
   command: >
-    {{ container_exec_cmd | default('') }} ceph --cluster ceph --name mon. -k
+    {{ hostvars[groups[mon_group_name][0]]['container_exec_cmd'] | default('') }} ceph --cluster ceph --name mon. -k
```

Comment:
Wouldn't doing this be easier - `{{ hostvars[groups[mon_group_name][0]]['container_exec_cmd'] | default('') }} ceph --cluster ceph --name mon. -k`?